vm/src/exceptions.rs

```diff
@@ -0,0 +1,40 @@
+
+use std::collections::HashMap;
+use super::pyobject::{PyFuncArgs, PyObject, PyObjectKind, PyObjectRef, PyResult};
+use super::vm::VirtualMachine;
+
+fn init(vm: &mut VirtualMachine, _args: PyFuncArgs) -> PyResult {
+    // TODO: Implement objint::str
+    Ok(vm.new_str("todo".to_string()))
```

Comment:
Init shouldn't return a value other than None. (type_call should enforce this but doesn't yet.)